prow/config.yaml

```diff
@@ -1,10 +1,7 @@
 ---
 plank:
-  # TODO(fejta): switch to spyglass
-  #ob_url_template: 'https://prow.istio.io/view/gcs/istio-prow/{{if eq .Spec.Type "presubmit"}}pull/{{.Spec.Refs.Org}}_{{.Spec.Refs.Repo}}/{{with index .Spec.Refs.Pulls 0}}{{.Number}}{{end}}{{else}}.{{end}}/{{.Spec.Job}}/{{.Status.BuildID}}/'
-  #ob_url_prefix: https://prow.istio.io/view/gcs
-  job_url_template: 'https://gubernator.k8s.io/build/istio-prow/{{if eq .Spec.Type "presubmit"}}pull/{{.Spec.Refs.Org}}_{{.Spec.Refs.Repo}}/{{with index .Spec.Refs.Pulls 0}}{{.Number}}{{end}}{{else}}.{{end}}/{{.Spec.Job}}/{{.Status.BuildID}}/'
-  job_url_prefix: https://gubernator.k8s.io/build/
+  job_url_template: 'https://prow.istio.io/view/gcs/istio-prow{{if eq .Spec.Type "presubmit"}}/pr-logs/pull/{{.Spec.Refs.Org}}_{{.Spec.Refs.Repo}}/{{with index .Spec.Refs.Pulls 0}}{{.Number}}{{end}}{{else if .Spec.Type "batch"}}/pr-logs/pull/batch{{end}}/{{.Spec.Job}}/{{.Status.BuildID}}'
```

Comment:
This omits the `org_repo` in the batch case. It also omits `/logs` in the periodic and postsubmit cases.

These templates are the worst :man_facepalming:

Reply:
Done

Although I'm pretty sure `org_repo` isn't in the batch case (see PR desc)